css/interface.css

```diff
@@ -1,3 +1,11 @@
+/* For Safari to hide scrollbar */
+html {
+  overflow: scroll;
+}
+::-webkit-scrollbar {
+  width: 0;
+  background: transparent; /* make scrollbar transparent */
```

Comment:
Why are we hiding scrollbars?

Reply:
In the Safari when we checked all items they are moving to the left for width of scrollbar.

Reply:
Can you demonstrate how this looks and behaves before and after your change?

Reply:
Before changes.
![with scrollbar](https://user-images.githubusercontent.com/29350776/59014906-6c71db00-8846-11e9-9675-992a26f79a60.png)
After changes.
![without scrollbar](https://user-images.githubusercontent.com/29350776/59014985-93301180-8846-11e9-8536-da56bf6912b7.png)

Reply:
Thanks. I wanted to make sure we were hiding the scrollbar when the scrollbar is necessary.

This looks good.